assignment5.js

```
@@ -0,0 +1,25 @@
+var fs = require('fs');
+
+var fileReader = function(fileName) {
+  return new Promise( (resolve, reject) => {
+    fs.readFile(fileName, 'utf8', function(error, data) {
+      console.log(data); 
+      console.log("============");
+
+      if(error)
+        return reject(fileName + 'file not Done');
+      resolve(fileName + 'fileDone');
+    });
+  });
+};
+
+console.log("============");
+
+fileReader('file1')
```

What will happen in case of an error. Error is not handled properly.